### ISSUE TITLE
Error while running Postgres 1.16.0-RC.1.sql migration

### ISSUE THREAD

**Issue by nscarlson** [FusionAuth/fusionauth-issues#606]
### Description 
Encounter an error while running the migration.  Beforehand, the database has been fully migrated to `1.15.8`.  

### Steps to reproduce
1. Ensure postgres database is migrated to 1.15.8 by running the `postgresql.sql` script from the direct downloads, following the advanced installation instructions.
2. Run the `1.16.0-RC.1` migration script
3. The migration fails with the error:

  ```
  ERROR:  invalid input syntax for type json
DETAIL:  Expected end of input, but found "}".
CONTEXT:  JSON data, line 1: ...ireMixedCase": false, "requireNonAlpha": false}}}
  ```

### Expected behavior
Expected the migration to run without issue.

### Platform
 - OS: MacOS Catalina 10.15.4
 - Postgres: version 11

**Comment by robotdan** [comment]
Thanks for the report @nscarlson  - we've also run into this issue. It will be fixed in the general availability release of 1.16.0.

**Comment by nscarlson** [comment]
Thank you @robotdan, I'll give that a try.
